lib/registerpage/registerpage_form.dart

```diff
@@ -0,0 +1,250 @@
+import 'dart:html';
+
+import 'package:flutter/material.dart';
+
+import 'registerpage_logo.dart';
+
+import '../constants/constants.dart' as Constants;
+
+import 'package:email_validator/email_validator.dart';
+
+class RegisterPageForm extends StatefulWidget {
+  final screenWidth;
+  final screenHeight;
+  const RegisterPageForm({Key? key, this.screenHeight, this.screenWidth}) : super(key: key);
+
+  @override
+  State<RegisterPageForm> createState() => _MyWidgetState();
+}
+
+class _MyWidgetState extends State<RegisterPageForm> {
+
+  final mailController = TextEditingController();
+  final passwordController = TextEditingController();
+  final secondPasswordController = TextEditingController();
+
+  var _outlineBorderColorMail = Color(0xffb9b9b9);
```

Comment:
Constants for colors, everywhere 😉 If you were to change the palette and also this particular color across the entire app, by having a constant you would have to change it in only 1 place and you will be sure it will change it everywhere.